### PR TITLE
Plfm 4198

### PR DESCRIPTION
**Pull request by john-hill**

### CODE REVIEW

File: integration-test/src/test/java/org/sagebionetworks/ITDownloadUsingFileHandleAssociationTest.java [hunk 1]
```diff
@@ -123,13 +123,20 @@ public void testWikiAttachment() throws SynapseException, IOException, Interrupt
...
 		assertNotNull(wiki);
-		
+		// get attachments
 		FileHandleAssociation fha = new FileHandleAssociation();
 		fha.setAssociateObjectId(wiki.getId());
 		fha.setAssociateObjectType(FileHandleAssociateType.WikiAttachment);
 		fha.setFileHandleId(fileHandle.getId());
 		URL url = synapse.getFileURL(fha);
 		assertNotNull(url);
+		// get markdown
```
Comment (kimyen) [inline integration-test/src/test/java/org/sagebionetworks/ITDownloadUsingFileHandleAssociationTest.java:133]: add a test for the manager instead of integration test

File: lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/wikiV2/V2DBOWikiPageDaoImplAutowiredTest.java [hunk 2]
```diff
@@ -1006,6 +1006,58 @@ public void testGetNumberOfVersions() {
...
+		two.setModifiedBy(creatorUserGroupId);
+		two.setMarkdownFileHandleId(attachOne.getId());
+		two.setAttachmentFileHandleIds(new LinkedList<String>());
+		two = wikiPageDao.create(two, new HashMap<String, FileHandle>(), ownerId, ownerType, new ArrayList<String>());
+		WikiPageKey twoKey = WikiPageKeyHelper.createWikiPageKey(ownerId, ownerType, two.getId());
+		toDelete.add(twoKey);
+		// call under test
+		List<String> fileHandleIds = Lists.newArrayList(markdownTwo.getId(), markdownOne.getId(), attachOne.getId());
+		Set<String> results = wikiPageDao.getFileHandleIdsAssociatedWithWikiMarkdown(fileHandleIds, one.getId());
+		assertNotNull(results);
```
Comment (kimyen) [inline lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/wikiV2/V2DBOWikiPageDaoImplAutowiredTest.java:1039]: add a test to test that Spring does not throw exception on empty result
